pkg/stores/sqlpartition/listprocessor/processor.go

```diff
@@ -53,6 +56,61 @@ type Cache interface {
 	ListByOptions(ctx context.Context, lo informer.ListOptions, partitions []partition.Partition, namespace string) (*unstructured.UnstructuredList, int, string, error)
 }
 
+func k8sOpToRancherOp(k8sOp selection.Operator) (informer.Op, bool, error) {
+	h := map[selection.Operator]informer.Op{
```

Comment:
Is it possible to define this map somewhere outside of here to avoid redefining it on each call?